hooks/use-debounced-effect.js

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useRef } from 'react'
+
+const useDebouncedEffect = (callback, deps = [], delay = 250) => {
```

Comment:
I think this function can be borrowed from some hooks package, there are [many of them](https://github.com/rehooks/awesome-react-hooks). Of course, I recommend to keep it under `./hooks` and reexport from here so we can support both our and external hooks from the single place.

Reply:
Surely if we need something complext. Currently it would be just another dependency to care. 🙂